lib/src/entrypoint.dart

```diff
@@ -743,7 +743,8 @@ class Entrypoint {
 
   /// Saves a list of concrete package versions to the `pubspec.lock` file.
   void _saveLockFile(SolveResult result) {
-    _lockFile = result.lockFile;
+    _lockFile =
```

Comment:
Can we add a comment that we use windows line-endings if the existing file already does so?

Reply:
Good idea!
Done